.github/workflows/vm_build.yaml

```diff
@@ -0,0 +1,125 @@
+name: Build and test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+
+jobs:
+  build-gvltctl:
+    name: Build gvltctl
+
+    # Matches test-vm-builder to avoid errors
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Update APT packages
+        run: |-
+          sudo apt-get update
```

Comment:
Is the intention here to only prepare for later package installations or actually upgrade all packages? `apt-get update` only updates the package index. `apt-get upgrade` upgrades packages themselves :slightly_smiling_face:

Reply:
Yep, just preparing for future installations.